sympy/geometry/ellipse.py

```diff
@@ -1268,6 +1268,55 @@ def vradius(self):
         return self.args[2]
 
 
+    def second_moment_of_area(self, point=None):
+        """Returns the second moment and product moment area of an ellipse.
+
+        Parameters
+        ==========
+
+        point : Point, two-tuple of sympifiable objects, or None(default=None)
+            point is the point about which second moment of area is to be found.
+            If "point=None" it will be calculated about the axis passing through the
+            centroid of the ellipse.
+
+        Returns
+        =======
+
+        I_xx, I_yy, I_xy : number or sympy expression
+                           I_xx, I_yy are second moment of area of an ellise.
+                           I_xy is product moment of area of an ellipse
```

Comment:
Does this render properly in the sphinx docs? The indentation shouldn't be this large.